pennylane/ops/qubit.py

```diff
@@ -1822,6 +1822,61 @@ def adjoint(self):
         return IsingXX(-phi, wires=self.wires)
 
 
+class IsingYY(Operation):
+    r"""IsingYY(phi, wires)
+    The Ising YY coupling gate
```

Comment:
```suggestion
    Ising YY coupling gate
```
For consistency with the other Ising XX and Ising ZZ operations